# 問題
>粗い床の上においた質量$5.0[kg]$のおもりを、水平となす角$30°$で引っ張ったところ$38[N]$を超えたところで動き出した。重力加速度
の大きさを$g=9.8[m/s2]$として、静止摩擦係数$μ$を求めよ。

In [16]:
from math import cos,sin,radians
import numpy as np
F = 38
m = 5.0
g = 9.8
angle = radians(30)
cosx = 0.866025 #cos(angle)
sinx = 0.5#sin(angle)

(F*cosx) / (m*g-F*sinx)

1.0969650000000002

解いていきます。

# 状況整理（力のつり合い）

おもり（質量 (m=5.0,\mathrm{kg})）に大きさ (F) の力を水平方向となす角 (30^\circ) 上向きに引く。
動き出す直前（静止限界）では

* 鉛直方向：法線力 (N) と重力 (mg)、引く力の上向き成分 (F\sin30^\circ) がつり合う
  [
  N + F\sin30^\circ = mg \quad\Rightarrow\quad N = mg - F\sin30^\circ
  ]
* 水平方向：引く力の水平成分 (F\cos30^\circ) と最大静止摩擦力 (\mu N) がつり合う
  [
  F\cos30^\circ = \mu N = \mu,(mg - F\sin30^\circ)
  ]

ここから (\mu) を解くと
[
\boxed{\ \mu=\dfrac{F\cos30^\circ}{,mg - F\sin30^\circ,}\ }
]

# 数値代入

条件より「(38,\mathrm{N}) を超えたところで動き出した」→ 限界はおよそ (F=38,\mathrm{N})。
(g=9.8,\mathrm{m/s^2}), (\sin30^\circ=\tfrac12), (\cos30^\circ=\tfrac{\sqrt3}{2}\approx0.866025), (mg=5.0\times9.8=49,\mathrm{N})。

* 分母：(mg - F\sin30^\circ = 49 - 38\times0.5 = 49 - 19 = 30,\mathrm{N})
* 分子：(F\cos30^\circ = 38\times0.866025 \approx 32.909,\mathrm{N})

したがって
[
\mu \approx \frac{32.909}{30} \approx 1.10
]

# 答え

[
\boxed{\mu \approx 1.10}
]

（角度をつけて上向きに引くと法線力が減る＝最大静止摩擦も減るため、同じ (F) でも水平方向の有効成分で見ると、(\mu) はやや大きめに見積もられます。今回の値はゴムと乾いた床などで起こり得る範囲です。）
